docs/reference/configuration/client/dfdaemon.md

```diff
@@ -206,6 +206,8 @@ proxy:
   # If the value is "true", the range request will prefetch the entire file.
   # If the value is "false", the range request will fetch the range content.
   prefetch: false
+  # prefetchRateLimit is the rate limit of the prefetch speed in KiB/MiB/GiB per second, default is 2GiB/s.
```

Comment:
prefetch_rate_limit is the rate limit of the prefetch speed in GiB/Mib/Kib per second. The prefetch request
has lower priority so limit the rate to avoid occupying the bandwidth impact other download tasks.